chia/util/file_keyring.py

```diff
@@ -51,9 +55,8 @@ def inner(self, *args, **kwargs):
 
 
 @contextmanager
-def acquire_writer_lock(lock_path: Path, timeout=5, max_iters=6):
+def acquire_writer_lock(lock_path: Path, timeout: int = 5, max_iters: int = 6) -> Iterator[None]:
     lock = fasteners.InterProcessReaderWriterLock(str(lock_path))
-    result = None
     for i in range(0, max_iters):
         if lock.acquire_write_lock(timeout=timeout):
```

Comment:
Unrelated to this PR, I'm a bit concerned about this.  If the body of the use of the context manager raises then the lock will not be explicitly released.  But, `fasteners` is going to be replaced anyways so maybe we take care of it then.  @paninaro @mariano54

Reply:
This code gets dropped in https://github.com/Chia-Network/chia-blockchain/pull/12541